packages/providers/src/transaction-summary/date.test.ts

```diff
@@ -0,0 +1,38 @@
+import * as tai64Mod from 'tai64';
```

Comment:
The tests are testing that libraries are being called, how many times and with which values, but not the actual conversion of values. Suggestion:

```suggestion
import { fromDateToTai64, fromTai64ToDate } from './date';

describe('transaction-summary/date', () => {
  const times = {
    tai64: '4611686020121838636',
    unix: 1694450695000,
  };
  it('should convert TAI64 to Date correctly', () => {
    const date = fromTai64ToDate(times.tai64);

    const expectedDate = new Date(times.unix);
    expect(date).toEqual(expectedDate);
  });

  it('should convert Date to TAI64 correctly', () => {
    const date = new Date(times.unix);

    const tai64 = fromDateToTai64(date);

    expect(tai64).toEqual(times.tai64);
  });
});
```

Reply:
@nedsalk It seems your review came only moments after I merged the PR. 

But yeah, since we are using an external lib I am not validating the actual values, only if the appropriate method was called.